.github/workflows/ci.yml

```diff
@@ -102,6 +102,51 @@ jobs:
       - name: Run tests
         run: |
           npm run test:ci
+    
+  ci-integration-test:
```

Comment:
Let's create an `integration.yml` file for this.

Reply:
@jsumners I don't have a strong opinion but I'm not sure that having this in a different workflow is better in general. A couple of main reasons:

- this workflow is the CI workflow. I think we want to run this new check conceptually as part of CI (meaning, any time there's a commit, any time there's a PR and so on)
- this workflow already has plenty of boilerplate that, since the new check will run pretty much under the same circumstances as CI, would have to be duplicated

I think that overall we should consider adding this check inside the existing CI workflow